src/databricks/labs/ucx/install.py

```diff
@@ -549,13 +553,21 @@ def _create_jobs(self):
                 self._state.jobs[step] = job_id
         logger.debug(f"Creating jobs from tasks in {main.__name__}")
         remote_wheel = self._upload_wheel()
+        self._ws.cluster_policies.edit(
+            policy_id=self._custom_cluster_policy_id,
```

Comment:
don't modify other people's cluster policies - it may be breaking their workloads